CHANGELOG.md

```diff
@@ -5,6 +5,37 @@
 ## [Unreleased]
 
 
+## [0.5.0]
+
+_2023-09-20_
+
+### Spark
+
+* 🆕 Add IconButtons with all intents, shapes and sizes
+* 🆕 Add loading state to the IconButton component
+* 🆕 Add icon toggle button
```

Comment:
nit: should we also use the proper composable name here? (or is it something else)
```suggestion
* 🆕 Add IconToggleButton
```